tests/Diagnostics.Tests/TestData/AutoHeal.csx

```diff
@@ -0,0 +1,442 @@
+#load "DetectorUtils"
```

Comment:
QQ - Can we write a test detector code here using gist and not use our actual detector code, to not accidentally leak any platform info (like table names etc)

Reply:
Sure,  I can write a sample one.